lib/peep.rb

```diff
@@ -0,0 +1,19 @@
+class Peep
+
+  def self.new_peep(message, user_id)
```

Comment:
Do we need a method for tags that belong to new peeps? what if we want to view all tags specific to a peep?